tests/python/gaia-ui-tests/gaiatest/apps/system/regions/cards_view.py

```diff
@@ -14,7 +14,7 @@
 class CardsView(Base):
 
     # Home/Cards view locators
-    _cards_view_locator = (By.ID, 'cards-view')
+    _task_manager_locator = (By.ID, 'task-manager')
```

Comment:
The cards view is a PageRegion of the system app. I think we can rename as the root_locator. What do you think?